oracle-plugin/src/e2e-test/resources/errorMessage.properties

```diff
@@ -1 +1,21 @@
 validationSuccessMessage=No errors found.
+errorMessageOracleInvalidReferenceName=Invalid reference name 'invalidRef&^*&&*'. Supported characters are: letters, numbers, \
+  and '_', '-', '.', or '$'.
+errorMessageBoundingQuery=Bounding Query must be specified if Number of Splits is not set to 1. Specify the Bounding Query.
+errorMessageBlankSplitBy=Split-By Field Name must be specified if Number of Splits is not set to 1. Specify the Split-by Field Name.
+errorMessageInvalidNumberOfSplits=Invalid value for Number of Splits '0'. Must be at least 1. Specify a Number of Splits no less than 1.
+errorMessageNumberOfSplitNotNumber=Unable to create config for batchsource Oracle 'numSplits' is invalid: Value of field\
+  \ class io.cdap.plugin.db.config.AbstractDBSpecificSourceConfig.numSplits is expected to be a number.
+errorMessageInvalidFetchSize=Invalid fetch size. Fetch size must be a positive integer.
+errorMessageInvalidSourceDatabase=SQL error while getting query schema: Listener refused the connection with the following \
+  error: ORA-12514, TNS:listener does not currently know of service requested in connect descriptor
+errorMessageInvalidImportQuery=Import Query select must contain the string '$CONDITIONS'. if Number of Splits is not set\
+  \ to 1. Include '$CONDITIONS' in the Import Query
+errorMessageBlankUsername=Username is required when password is given.
+errorMessageInvalidTableName=Table 'invalidTable' does not exist. Ensure table '"invalidTable"' is set correctly and that the\
+  \ connection string 'jdbc:oracle:thin:@//35.222.99.157:1521/xe' points to a valid database.
```

Comment:
we should not be pushing actual host addresses to public github, please fix.

Reply:
Done.